src/Protector.php

```diff
@@ -132,36 +134,41 @@ public function importDump(string $sourceFilePath, array $options): bool
         Storage::disk('local')->put($sourceFilePath, $this->getDisk()->get($sourceFilePath));
         $filePath = Storage::disk('local')->path($sourceFilePath);
 
-        try {
-            $shellCommandDropCreateDatabase = sprintf('mysql -h%s -u%s -p%s -e %s 2> /dev/null',
-                escapeshellarg($this->connectionConfig['host']),
-                escapeshellarg($this->connectionConfig['username']),
-                escapeshellarg($this->connectionConfig['password']),
-                escapeshellarg(sprintf('drop database %1$s; create database %1$s;', $this->connectionConfig['database'])));
+        $this->isExecEnabled();
```

Comment:
Why would we generate all those shell calls, when it is impossible to execute them.

**Part 2**

Reply:
Fixed.